drivers/syslog/syslog_filechannel.c

```diff
@@ -60,6 +60,23 @@ FAR static struct syslog_channel_s *g_syslog_file_channel;
  * Private Functions
  ****************************************************************************/
 
+#ifdef CONFIG_SYSLOG_FILE_SEPARATE
+static void log_separate(FAR const char *log_file)
+{
+  int fd;
+
+  fd = open(log_file, O_WRONLY);
```

Comment:
change to file_open/file_write/file_close

Reply:
Oops, forgot this again...

Shall I use `file_open()` or `nx_open()`?

Reply:
Yes, the prefer is file_open->nx_open->open inside kernel space. Kernel space perfer file_xxx, but since not all fs function has the kernel version, we have to fallback to nx_ or userspace version.

Reply:
OK, done!